messaging/src/main/java/org/axonframework/messaging/annotation/MethodInvokingMessageHandlingMember.java

```diff
@@ -173,7 +173,7 @@ public MessageStream<?> handle(@Nonnull Message<?> message,
             if (e.getCause() instanceof Exception) {
                 return MessageStream.failed(e.getCause());
             } else if (e.getCause() instanceof Error) {
-                throw (Error) e.getCause();
+                return MessageStream.failed(e.getCause());
```

Comment:
Hmmm... Now I'm not sure about that. 
I thought we do not handle `Exception` at all. 
Should we handle things like `OutOfMemoryError`?